### ISSUE TITLE
Style enhancement part not used

### ISSUE THREAD

**Issue by FlorentF9** [Flashkong/Source-Free-Object-Detection-by-Learning-to-Overlook-Domain-Style#6]
Hi,

I noticed that in the code provided in this repo, the style enhancement module is actually never used. In `Overlook/train_target.py`, line 594 there is:

`im_data2 = im_data.detach().clone()`

meaning that the same input is provided to student and teacher, without random augmentation, style transfer or style enhancement. 
Also all command-line parameters related to style are unused (random_style, style_add_alpha, etc.).

Obviously, this doesn't allow to reproduce the results from the paper.

Could you consider to update your code to be able to use your method?
Thanks!

**Comment by Flashkong** [comment]
Thank you for your interest in our work!

The style enhancement part is used in the forward function of faster rcnn. In `Overlook/lib/model/faster_rcnn/faster_rcnn_style.py`, line 66-68:

```
if self.training and target:
    if student:
        im_data = im_data*0.0+1.0 * self.adain.add_style(im_data, 0)
```

The add ratio is 1.0, which ensures that the image fed into the student model is an style enhanced image.

**Comment by FlorentF9** [comment]
Thanks for your answer! I had finally figured it out by digging in the code.
